CONTRIBUTING.md

```diff
@@ -8,13 +11,18 @@ There might be a little voice inside that tells you you're not ready; that you n
 
 I assure you, that's not the case.
 
-This project has some clear Contribution Guidelines and expectations that you can read here (link).
+### Contributing Guidelines
+
+We have set some clear contribution guidelines to make it easier for you to get involved. You can find these guidelines [here]().
```

Comment:
Did you want a link here?

Reply:
This is only the placeholder for the link to the guidelines (file does not exist yet)